PABC.AppHost/Program.cs

```diff
@@ -7,6 +7,7 @@
 var postgresdb = postgres.AddDatabase("Pabc");
 
 var migrations = builder.AddProject<Projects.PABC_MigrationService>("migrations")
+    .WithEnvironment("JSON_DATASET_PATH", "../test-dataset.json")
```

Comment:
Wordt hij nu dan altijd geladen? wat als je geen dataset wil importeren?

Ik zou hier wel wat instructies bij verwachten in de documentatie. Ik zie niet direct in de openstaande taken terug of dat nog komt.

Reply:
Als je een dataset specificeert, wordt deze bij elke deployment ingeladen en overschrijft deze de bestaande data. Dit is voor ZAC in eerste instantie voldoende. Groot gelijk over de documentatie. Ik zal een taak toevoegen

Reply:
done